src/main/java/org/ibp/api/java/impl/middleware/dataset/validator/StudyValidator.java

```diff
@@ -55,7 +55,7 @@ public void validate(final Integer studyId, final Boolean shouldBeUnlocked) {
 
 		if (shouldBeUnlocked
 			&& study.isLocked()
-			&& !ObjectUtils.equals(study.getCreatedBy(), String.valueOf(contextUtil.getIbdbUserId(loggedInUser.getUserid())))
+			&& !study.getCreatedBy().equals(loggedInUser.getUserid().toString())
```

Comment:
Interesting, I think I've used `ObjectUtils.equals` here in case `createdBy` was null, but it's deprecated now in favor of `java.util.Objects.equals`, perhaps we should use that instead if the reference can be null, though it's not likely here